Lib/glyphsLib/builder/transformations/propagate_anchors.py

```diff
@@ -299,6 +299,7 @@ def origin_adjusted_anchors(anchors: list[GSAnchor]) -> Iterable[GSAnchor]:
         GSAnchor(
             name=a.name,
             position=Point(a.position.x - origin.x, a.position.y - origin.y),
+            userData=a.userData,
```

Comment:
since we want to copy the anchors and this is a mutalbe object (a dict or some custom dict-like object, I don't remember?), it would be better to actually make a copy of the userData as well, so the two anchors are entirely independent from one another, and if one gets modified the other doesn't accidentally